src/test/java/christmas/ApplicationTest.java

```diff
@@ -49,6 +49,298 @@ class ApplicationTest extends NsTest {
         });
     }
 
+    @Test
+    void 날짜_범위_예외_테스트() {
+        assertSimpleTest(() -> {
+            runException("32");
+            assertThat(output()).contains("[ERROR] 유효하지 않은 날짜입니다. 다시 입력해 주세요.");
+        });
+    }
+
+    @Test
+    void 날짜_범위_예외_테스트2() {
+        assertSimpleTest(() -> {
+            runException("0");
+            assertThat(output()).contains("[ERROR] 유효하지 않은 날짜입니다. 다시 입력해 주세요.");
+        });
+    }
+
+    @Test
+    void 날짜_입력_예외_테스트() {
```

Comment:
단순히 `~~예외 테스트` 라고만 이름 짓는 것 보다는 어떤 예외상황인지를 조금 더 자세히 적어준다면 테스트 코드가 실패했을 때 트러블 슈팅이 편해집니다!
```suggestion
    @DisplayName("숫자 이외에 다른 값이 함께 입력되면 예외가 발생한다.")
    void 날짜_입력_예외_테스트() {
```